conftest.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+
+try:
+    import sybil
+    import sybil.parsers.rest
+
+except ImportError:
+    sybil = None
+
+import lammpsio
+
+try:
+    import gsd.hoomd
+
+    has_gsd = True
+except ModuleNotFoundError:
+    has_gsd = False
+
+
+def setup_sybil_tests(namespace):
+    """Sybil setup function."""
+    # Common imports.
+    namespace["numpy"] = numpy
+    namespace["lammpsio"] = lammpsio
+    if has_gsd:
+        namespace["frame"] = gsd.hoomd.Frame()
+    else:
+        namespace["frame"] = 0
```

Comment:
Why `0` and not `None`? And why create the `Frame()` here, rather than where it's needed?

I would think you would want to optionally bind `"gsd"` or `"gsd.hoomd"` in the namespace, and set it to `None` if the import failed.